Packs/Phishing/Playbooks/Phishing_-_Generic_v3.yml

```diff
@@ -1866,7 +1865,7 @@ tasks:
                 iscontext: true
               right:
                 value:
-                  simple: ReporterAddress
+                  simple: incident.reporteremailaddress
```

Comment:
The field `incident.reporteremailaddress` will hold the value of `ReporterAddress` since it is being set in task 85. `ReporterAddress` is in fact holding the sanitized value (task 134 + 12).
I can see you moved this task (85) to the top of the playbook which seems logical, however, I don't see the difference with using either one of `ReporterAddress` and `incident.reporteremailaddress` since both will hold the same value.
`ReporterAddress` is in fact holding the sanitized value.

Reply:
oh you're actually right.. so this makes the PR pointless

Reply:
I might introduce this (logical) change in another PR with another fix, but I'll close this PR